extensions/dacpac/src/localizedConstants.ts

```diff
@@ -67,3 +67,7 @@ export const reservedValueErrorMessage = localize('dacfx.reservedValueErrorMessa
 export const trailingWhitespaceErrorMessage = localize('dacfx.trailingWhitespaceErrorMessage', "File name cannot end with a whitespace");
 export const tooLongFilenameErrorMessage = localize('dacfx.tooLongFilenameErrorMessage', "File name is over 255 characters");
 export function deployPlanErrorMessage(errorMessage: string): string { return localize('dacfx.deployPlanErrorMessage', "Generating deploy plan failed '{0}'", errorMessage ? errorMessage : 'Unknown'); }
+export function deployOperationErrorMessage(errorMessage: string): string { return localize('dacfx.deployOperationErrorMessage', "Deploy operation failed '{0}'", errorMessage ? errorMessage : 'Unknown'); }
```

Comment:
these can be consolidated into one function by also passing in the operation name. There are already strings for each operation above.